script/update_sanctions_csv

```diff
@@ -6,8 +6,9 @@ use File::Temp qw/tempfile/;
 use Data::Validate::Sanctions;
 
 die "LWP::UserAgent is required for download csv." unless eval("require LWP::UserAgent;");
-die "Text::CSV is required for parsing csv." unless eval("require Text::CSV;");
+die "Text::CSV is required for parsing csv."       unless eval("require Text::CSV;");
```

Comment:
These are in prereqs already, so no need for the checks - you can just `use` them directly.